.github/CODEOWNERS

```
@@ -0,0 +1,31 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the repo.
+* @pierre-sassoulas @DanielNoord @cdce8p @jacobtylerwalls @hippo91
+
+# Order is important. The last matching pattern has the most precedence.
+
+# @DudeNr33
+pylint/pyreverse/*    @DudeNr33
+tests/pyreverse/*    @DudeNr33
+
+# @pierre-sassoulas
+pylint/messages/*    @pierre-sassoulas
+
+# typing
+pylint/typing.py     @DanielNoord @cdce8p
+
+# multiprocessing
+# pylint/lint/parallel.py @doublethefish
+# tests/test_check_parallel.py @doublethefish
+
+# @yushao2
+tests/functional/u/unused/unused_private_member.py @yushao2
+
+# @areveny
+pylint/extensions/for_any_all.py @areveny
+tests/functional/ext/for_any_all/for_any_all.py @areveny
```

```suggestion
pylint/extensions/for_any_all.py @areveny
tests/functional/ext/for_any_all/* @areveny
```
I like that pattern, especially for extensions. We should just use a wildcard for the tests directory.